### ISSUE TITLE
Is this still active?

### ISSUE THREAD

**Issue by dhwang** [developerforce/salesforce-canvas-seed#1]
If not, what is the replacement?

**Comment by dcarroll** [comment]
It's a sample app for getting started with Play and Java. Granted it hasn't been updated in a while, but I don't think it needs to be.

@jamesward This still is current?

**Comment by jamesward** [comment]
Yeah, it should still work and I still support it.  It goes along with: https://www.jamesward.com/2015/03/10/salesforce-canvas-quick-start-for-java-developers
